contracts/PatriciaTree/PatriciaTreeProofs.sol

```diff
@@ -50,7 +50,7 @@ contract PatriciaTreeProofs {
       uint bit;
       (bit, e.label) = e.label.chopFirstBit();
       edgeHashes[bit] = e.edgeHash();
-      edgeHashes[1 - bit] = siblings[siblings.length - i - 1];
+      edgeHashes[1 - bit] = siblings[siblings.length - i - 1]; // ignore-swc-101
```

Comment:
Looking at this now I don't know why the `for` loop was done this way and not `for (uint i = 1; i <= siblings.length; i++)`, then the `i-1` would just be `i`.
Also is this asking us to `assert(siblings.length<=siblings.length - i - 1)`?